File: Python Word Examples/TextRangesApiDemo/getTextRanges.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.TextRangesApi(configuration)

file_name = 'Template.docx'
extra = {'folder': "input"}
paragraph_path = 'Section/0/Body/0/Paragraph/0'
result = api.get_text_ranges(file_name, paragraph_path, **extra)

File: PHP Word Examples/TextRangesApiDemo/getTextRanges.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\TextRangesApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new TextRangesApi($configuration);

$fileName = "Template.docx";
$paragraph_Path = "Section/0/Body/0/Paragraph/0";
$folder = 'input';
$storage = null;
$password = null;
$result = $apiInstance->getTextRanges($fileName, $paragraph_Path, $folder, $storage, $password);